python-backend/app/commands.py

```diff
@@ -162,3 +163,22 @@ def delete_data():
             db.session.rollback()
             click.echo(f'Error, failed on commit.')
             raise
+
+    #Had to define this function twice since the annotation used is different based on the environment.
```

Comment:
Is there a reason not to put the scheduled tasks in their own file? The rest of these commands are meant for development only to manage mock seed data. This command serves an entirely different concern.

Also, why is it necessary to provide this method at all in a non-production environment? If someone tried to run this in a test or dev instance, I'm sure a 'method not defined' error would be acceptable.

Reply:
I don't really see a reason to put them in separate files if we had a ton of commands registered to the app I could see splitting them up but at the moment we only have a handful. As for the If statement I have removed it as it was needed and then became unnecessary so I have removed it.

Reply:
I would argue that even just two commands for entirely different environments/purposes would still warrant two separate files.

What's the harm in having a separate scheduled tasks or production tasks file?

Reply:
I'm unsure what you asking for here. There are separate files for the actual tasks this is just a flask command being registered so that they can be force started from the pods CLI. This command doesn't have anything to do with scheduling.

Reply:
My mistake. I thought this was your ETL automation PR.

Unless I'm totally missing something here, my point still stands about having a file for production-only commands, though, no?

Reply:
I'd rather see all the commands in the commands.py files, the condition (or a comment) explains the intent. If these commands start growing any sort of interdependence then maybe it makes sense to break them down, i think it's unnecessary to have multiple 'commands' files right now.

Reply:
I don't see a reason to split up the commands at the moment, there are so few of them that I don't think adding another file will benefit us.